internal/generator/suite.go

```diff
@@ -40,9 +41,28 @@ func (s *Suite) SetupSuite() {
 	{{ end }}
 	{{ .Cleanup }}
 	{{ .Run }}
+
+	s.RunIncludedSuites()
+}
+
+func (s *Suite) RunIncludedSuites() {
+	{{ .TestIncludedSuites }}
```

Comment:
We should not generate this if there are no included suits. Consider looking at go-template conditions.

Reply:
Done